ChangeLog.md

```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 For a detailed look at the project's future, including planned features and bug
 fixes, check out the
 [roadmap](https://github.com/goatshriek/stumpless/blob/master/docs/roadmap.md).
-
+### Added 
+ - @since format check enforcement in CI pipeline.
```

Comment:
This should go under the 2.2.0 heading, instead of above it

Reply:
ohh i saw the unreleased tag and assumed thats where it should go. should i remove that section since there is no next/staging branch?

should the date be updated aswell on the 2.2.0 tag?

Reply:
You're right, the "Unreleased" heading is a little confusing - it's a holdover from an earlier changelog that also included the roadmap content. Feel free to remove the heading (line 7) itself, just make sure you leave the paragraph and link to the roadmap in.

Ideally please update the 2.2.0 tag date as well, though this is only critical when a release is cut, which will likely be sometime in November for 2.2.0.

Don't worry too much about trying to do force pushes or amends - the pull will be squashed when it's merged.

Reply:
done, moved the entry and removed unrelease tag.